corehq/apps/geospatial/static/geospatial/js/geospatial_map.js

```diff
@@ -433,6 +433,22 @@ hqDefine("geospatial/js/geospatial_map", [
         mapModel.fitMapBounds(caseMapItems);
     }
 
+    function sanitizeDisbursementParametersText(parameters) {
+        var sanitizedParameters = [];
+
+        parameters.forEach(function (param) {
+            var sanitizedParam = {name: param.name, value: gettext("Unspecified")};
+            if (param.value) {
+                sanitizedParam.value = `${param.value}`;
+                if (param.unit) {
+                    sanitizedParam.value = sanitizedParam.value + ` ${param.unit}`;
```

Comment:
Nit: Since you are already using a templated string here you can include the value var as well with `` `${sanitizedParam.value} ${param.unit}` ``

Reply:
I initially did that, but then it's formatted as "value " (note the additional space at the end). I just don't like this.